hacks/build-baremetal-installer.md

```diff
@@ -0,0 +1,87 @@
+# How to build the openshift-baremetal-install binary in RHEL8
+
+Those instructions are intended to show how to compile the `openshift-baremetal-install`
+from scratch in RHEL8 in case it is required to apply some PRs:
+
+* Install golang and required libs
+
+```shell 
+sudo yum install golang-bin gcc-c++ libvirt-devel
+```
+
+* Create some folders required for golang and to store the openshift-install binary
+
+```shell
+mkdir -p ~/go/{src,bin}
+mkdir -p ~/bin/
+```
+
+* Setup the environment
+
+```shell
+export GOPATH=${HOME}/go/
+export PATH="${GOPATH}/bin:${HOME}/bin:$PATH"
+```
+
+* Get the sources
+
+```shell
+go get -v -u github.com/openshift/installer
+go get -v -u github.com/golang/dep
+```
+
+* Build dep
+
+```shell
+go install ${GOPATH}/github.com/golang/dep
+```
+
+* Configure fake user/email in order to allow git am
+
+```shell
+pushd ${GOPATH}/github.com/openshift/installer
+git config user.name foo
+git config user.email foo@bar.com
+```
+
+* Pull latest just in case
+
+```shell
+git pull
+```
+
+* Download deps
+
+```shell
+dep ensure -v
+```
+
+* Apply PRs
+
+```shell
+for pr in 1 2 3; do
+  curl -L https://github.com/openshift/installer/pull/${pr}.patch | git am
+done
+```
+
+* Build the binary
+
+```shell
+TAGS="baremetal libvirt" hack/build.sh
+```
+
+* Copy the binary where it belongs
+
+```shell
+cp bin/openshift-install ${HOME}/bin/
```

Comment:
uhm, is ${HOME}/bin guaranteed to be in ${PATH}? otherwise this step isn't very useful

Reply:
Is is exported before (`export PATH="${GOPATH}/bin:${HOME}/bin:$PATH"`) but I can add a clarification that this is an optional step.